nominations/models.py

```diff
@@ -192,58 +218,180 @@ class Application(models.Model):
         choices=primary_election_type_choices,
         null=True,
     )
-    questionnaire = models.ForeignKey(Questionnaire, on_delete=models.SET_NULL, null=True, blank=True)
+    questionnaire = models.OneToOneField(
```

Comment:
Are you sure this is a safe migration on prod db?

Reply:
From a data standpoint yes, because a OneToOneField in Django is like a ForeignKey with `unique=True` based on its implementation.

There is one other difference, and that is a foreign key returns a queryset when trying to reference it's attached objects (i.e. `application_set`) whereas a one-to-one field will return a single object. This is the one issue we may run into with A/B deploy - if code is referencing `questionnaire.application_set`, we might run into a temporary issue. This is used only here - https://github.com/Our-Revolution/site/blob/207b2515040d53a98f36d7fb7f3757d1799300e2/nominations/models.py#L124.

Am thinking through A/B deploy issues now.

Reply:
@cjmabry if the migrations add a uniqueness constraint that wasn't there before, it might throw error if prod has duplicate data already in table?

Thanks for checking about A/B deploy issues. Also this seems out of scope for this ticket, so I'm not sure why this schema change is happening here? Might be best to back it out at this point if not needed for ticket.

Reply:
You're right, I need sleep.

Removing!